plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/main/java/org/eclipse/che/jdt/refactoring/RefactoringManager.java

```diff
@@ -326,4 +331,17 @@ public void changeChangeEnabled(ChangeEnabledState state) throws RefactoringExce
         RefactoringSession session = getRefactoringSession(state.getSessionId());
         session.updateChangeEnabled(state.getChangeId(), state.isEnabled());
     }
+
+    public ChangePreview getChangePreview(RefactoringChange change) throws RefactoringException, CoreException {
+        RefactoringSession session = getRefactoringSession(change.getSessionId());
+        PreviewNode previewNode = session.getChangePreview(change.getChangeId());
+        ChangePreviewViewerDescriptor descriptor = previewNode.getChangePreviewViewerDescriptor();
+        if(descriptor != null){
```

Comment:
This code is not formatted, space required before left brace <code>) {</code> and after <code>if</code>.
Also you can avoid nesting by changing negative statement to positive:
```java
if (descriptior == null) {
    return null
}
```